src/yggdrasil/nodeinfo.go

```diff
@@ -108,9 +108,14 @@ func (m *nodeinfo) setNodeInfo(given interface{}) error {
 	if nodeinfomap, ok := given.(map[string]interface{}); ok {
 		for key, value := range nodeinfomap {
 			if _, ok := newnodeinfo[key]; ok {
+				if value == "hide" {
```

Comment:
should `strings.EqualFold`, to allow any case of "Hide" "HIDE" etc.
also would this even compile, as value will probably be of type `interface{}` not `string`?